### PR TITLE
[Fix] UNO 588 - Unexpected scroll bar appears

### PR DESCRIPTION
**Pull request by habemuscode**
**Related to**

https://oat-sa.atlassian.net/browse/UNO-588

**Description**

When user adds any interaction, goes to response and tries to hover on any tooltip unexpected scroll bar appears.

**Steps to reproduce**

1. Login as admin
2. Go to Items tab
3. Create a new item and go to Authoring
4. Add any interaction to the canvas (f.e. choice interaction)
5. Go to Response and try to hover any tooltip in the response properties

### CODE REVIEW

**Review by bziondik — APPROVED** [review]
- [ ]  New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [ ]  New code is not subject to concurrency issues (if applicable)
- [x]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

File: scss/inc/_select2.scss [hunk 1]
```diff
@@ -49,7 +49,7 @@
 }
 
 .select2-container .select2-choice {
-    display: block;
+    display: flex;
```
Comment (andrey-ponamarev) [inline scss/inc/_select2.scss:52]: Have you tried to add `position: relative` to this selector?
The revert of line 56 should fix it https://github.com/oat-sa/tao-core-ui-fe/commit/8e999ca987b194f0e8398b4e3766a48926676268#diff-9ddf7e56610168b3d70cd82d877989af

**Review by andrey-ponamarev — CHANGES_REQUESTED** [review]

**Review by AndreyNegrov — APPROVED** [review]
- [ ]  New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [ ]  New code is not subject to concurrency issues (if applicable)
- [ ]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

File: scss/inc/_select2.scss (hunk 1 above)
Comment (habemuscode) [reply scss/inc/_select2.scss:52]: I try but then create the bug that I fix time ago about scroll

https://github.com/oat-sa/tao-core-ui-fe/pull/165

**Pull request merged**